config/packages/security.yaml

```diff
@@ -0,0 +1,44 @@
+security:
+    # https://symfony.com/doc/current/security.html#where-do-users-come-from-user-providers
+    providers:
+        # used to reload user from session & other features (e.g. switch_user)
+        app_user_provider:
+            entity:
+                class: App\Entity\User
+                property: email
+
+    firewalls:
+        dev:
+            pattern: ^/(_(profiler|wdt)|css|images|js)/
+            security: false
+        main:
+            anonymous: ~
+            guard:
+                authenticators:
+                    - App\Security\LoginFormAuthenticator
+            logout:
+                path:   app_logout
+
+            # activate different ways to authenticate
+
+            # http_basic: true
+            # https://symfony.com/doc/current/security.html#a-configuring-how-your-users-will-authenticate
+
+            # form_login: true
+            # https://symfony.com/doc/current/security/form_login_setup.html
+
+    # Easy way to control access for large sections of your site
+    # Note: Only the *first* access control that matches will be used
+
+    access_control:
+        - { path: ^/organisations, roles: ROLE_USER }
+        - { path: ^/sponsorship-level, roles: ROLE_USER }
+        - { path: ^/sponsorship-level-benefit, roles: ROLE_USER }
+        - { path: ^/speakers, roles: ROLE_USER }
+        - { path: ^/special-benefit, roles: ROLE_USER }
+        - { path: ^/sponsorship-benefit, roles: ROLE_USER }
+
+    encoders:
+        App\Entity\User:
+            algorithm: bcrypt
```

Comment:
Why not [argon2i with libsodium](https://symfony.com/blog/new-in-symfony-3-4-argon2i-password-hasher)?